File: week09/week09-3.py
class Solution:
    def average(self, salary: List[int]) -> float:
        salary.sort() #先從小到大排好

        total=0
        for i in range(1,len(salary)-1): #避開最左、右邊
            total+=salary[i]
        return total/(len(salary)-2) #平均

File: week09/week09-3.cpp
class Solution {
public:
    double average(vector<int>& salary) {
        sort(salary.begin(),salary.end()); //從小到大排好
        double total=0; //有小數點
        for(int i=1;i<salary.size()-1;i++){
            total+=salary[i];
        }//避開最小和最大
        return total/(salary.size()-2); //平均
    }
};